x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx

```diff
@@ -42,16 +43,30 @@ const i18nTexts = {
   }),
   pageDescription: i18n.translate('xpack.upgradeAssistant.esDeprecations.pageDescription', {
     defaultMessage:
-      'You must resolve all critical issues before upgrading. Back up recommended. Make sure you have a current snapshot before modifying your configuration or reindexing.',
+      'You must resolve all critical issues before upgrading. Back up recommended. Make sure you have a current snapshot before modifying your configuration or reindexing. ',
```

Comment:
Unless I'm missing something, making a backup and taking a snapshot are the same thing. I think we can also have a stronger lead-in for reindexing.
```suggestion
      'Resolve all critical issues before upgrading. Before making changes, ensure you have a current snapshot of your cluster. Some issues may require reindexing to resolve. ',
```